zetaclient/metrics/telemetry.go

```diff
@@ -50,6 +53,18 @@ func NewTelemetryServer() *TelemetryServer {
 	return hs
 }
 
+func (t *TelemetryServer) SetConnectedPeers(peers []peer.AddrInfo) {
+	t.mu.Lock()
+	t.connectedPeers = peers
+	t.mu.Unlock()
+}
+
+func (t *TelemetryServer) GetConnectedPeers() []peer.AddrInfo {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	return t.connectedPeers
```

Comment:
Above we do
```
	t.mu.Lock()
	t.connectedPeers = peers
	t.mu.Unlock()
```
and here we use defer which is a bit inconsistent, we should do same everywhere